portfolio/src/main/webapp/aboutme.html

```diff
@@ -0,0 +1,77 @@
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+
+
+    <title>Chiamaka Chukwuka</title>
+
+
+    <link href="https://fonts.googleapis.com/css?family=Princess+Sofia" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Miss+Fajardose" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Ewert" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Libre+Barcode+39+Extended+Text" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Rock+Salt" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Cinzel+Decorative" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Poiret+One&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet" type="text/css" />
+
+
+  </head>
+  <body>
+
+
+    <div id="content">
+
+
+    <div id="logo"><em><h1 id="first"><div id="altletters"><span id="c1">C</span><span id="c12">hi</span><span id="c2">&#10037;a<span id="c23">&#10037;</span></span><span id="c3">m</span><span id="c4">a</span><span id="c45">&#10037;</span><span id="c5">k</span><span id="c6">a</span></div></h1></em></div>
+    <h3 id="subhead"> &#9830; Chiamaka  (Chee-uh-mok-uh)  Chukwuka  (Chew-kwoo-kah). &#9830;</h3>
+    <script src="script.js"></script>
+
+    
+    </div>
+    <head><title>Artwork</title></head>
+    <body>
+      <link href="https://fonts.googleapis.com/css?family=Playfair+Display+SC" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/css?family=Homemade+Apple" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/css?family=Monsieur+La+Doulaise" rel="stylesheet">    
+      <link href="gala.css" rel="stylesheet" type="text/css" />
```

Comment:
I don't see the gala.css file in Github, did you `git add` it?

Reply:
It's supposed to be style.css I'll fix it. 
I used the google fonts tool

Reply:
Resolving because I fixed the incorrect naming of style sheet